sauron-service/plugins/plugins.json

```diff
@@ -0,0 +1,167 @@
+[
+  {
+    "id": "jaegerapi-report",
+    "description": "jaegerapi-report",
+    "releases": [
+      {
+        "version": "0.0.3",
+        "date": "2021-03-29T10:23:51Z",
+        "url": "https://repo1.maven.org/maven2/com/free-now/sauron/plugins/jaegerapi-report/0.0.3/jaegerapi-report-0.0.3.jar"
```

Comment:
I guess the url is different than the one configured in the pom.xml files, isn't it?

Reply:
> I guess the url is different than the one configured in the pom.xml files, isn't it?

This is correct, we are interested only in the jar published in maven central.